website/components/Collapse.scss

```diff
@@ -0,0 +1,14 @@
+.ant-collapse-header {
+  padding: 6px 10px !important;
```

Comment:
the original padding of antd looks better..

Reply:
Too big and will use too much space